Wikipedia/Code/PageHistoryCollectionViewCell.swift

```diff
@@ -52,8 +55,19 @@ class PageHistoryCollectionViewCell: CollectionViewCell {
         }
     }
 
+    func updateAccessibilityLabel() {
+        let isMinorAccessibilityString = isMinor ? WMFLocalizedString("page-history-revision-minor-edit-accessibility-label", value: "Minor edit", comment: "Accessibility label text used if edit was minor") : ""
+        accessibilityLabel = [timeLabel.accessibilityLabel, authorButton.accessibilityLabel, sizeDiffLabel.accessibilityLabel, isMinorAccessibilityString, commentLabel.accessibilityLabel]
+            .compactMap { $0 }
+            .filter { $0.wmf_hasNonWhitespaceText }
+            .joined(separator: ", ") // Comma adds slight voice-over pause.
+    }
+    
     var comment: String? {
         didSet {
+            if let comment = comment, comment.wmf_hasNonWhitespaceText {
```

Comment:
Same comment as above - if it needs to be nil'd out for whitespace add an `else { comment.accessoryLabel = nil }`

Reply:
fixed!